### PR TITLE
Fix wording and links lifecycle tutorial

### PR DESCRIPTION
**Pull request by oscargus**
## PR Summary

There was an incorrect sentence that needed rewording. Links added for easier navigation. Add a ,. Replace 1 with one.
## PR Checklist

**Tests and Styling**
- [ ] Has pytest style unit tests (and `pytest` passes).
- [ ] Is [Flake 8](https://flake8.pycqa.org/en/latest/) compliant (install `flake8-docstrings` and run `flake8 --docstring-convention=all`).

**Documentation**
- [ ] New features are documented, with examples if plot related.
- [ ] New features have an entry in `doc/users/next_whats_new/` (follow instructions in README.rst there).
- [ ] API changes documented in `doc/api/next_api_changes/` (follow instructions in README.rst there).
- [ ] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).

### CODE REVIEW

**Review by timhoffm — APPROVED** [review]

File: tutorials/introductory/lifecycle.py [hunk 1]
```diff
@@ -26,24 +26,24 @@
...
 pyplot interface.
 
 Most of the terms are straightforward but the main thing to remember
 is that:
 
-* The Figure is the final image that may contain 1 or more Axes.
-* The Axes represent an individual plot (don't confuse this with the word
-  "axis", which refers to the x/y axis of a plot).
+* The `.Figure` is the final image, and may contain one or more `~.axes.Axes`.
+* The `~.axes.Axes` represents an individual plot (not to be confused with
```
Comment (timhoffm) [inline tutorials/introductory/lifecycle.py:38]:
```suggestion
* The `~.axes.Axes` represents an individual plot (not to be confused with
```
To placate flake8

Comment (oscargus) [reply tutorials/introductory/lifecycle.py]: Thanks! Have not had time to see where it came from.

**Comment by oscargus** [comment]
Please squash-merge...

**Comment by rcomer** [comment]
@meeseeksdev backport to v3.6.x

**Pull request merged**